tsconfig.test.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "lib": ["es2020", "dom"]
+  },
+  "include": ["**/*.test.ts"],
+  "exclude": ["node_modules"]
```

Comment:
Nit: I believe `node_modules` are excluded by default so we might not need this

Reply:
Correct, it's more that I needed to override `exclude` because I was getting a complaint that `**/*.test.ts` was in both include and exclude.

Reply:
Makes sense

Reply:
I removed `node_modules` and just left `[]` and everything is working.